### PR TITLE
WCAG updates in Compliance with Axe API

### PR DESCRIPTION
**Pull request by 21shield**
### Ticket
closes # Axe api error alerts found on Home Component

### Checklist
- [x] There are no warnings in the browser console
- [x] I made sure the component is WCAG compliant using ANDI
- [x] I made sure the UI follows the style guide in the wireframe (font, font-size, etc...)
- [x] I made sure that the screen reader can read all the text
- [x] I made sure I removed all the console.logs
- [x] I made sure mobile portrait view looks good
- [x] I made sure mobile landscape view looks good
- [ ] I added storybook for the component
	- [x] Storybook not applicable
- [ ] I added unit test coverage
	- [ ] I added integration test coverage
	- [x] Unit test coverage not applicable

### To Do
- [ ] Review Color change for font of buttons
- [ ] Create a default main element for most pages
- [ ] Review with Team

### Additional Notes
There is a commented our color choice for a font color that needs to be reviewed.

### CODE REVIEW

File: src/components/Home/Home.js [hunk 1]
```diff
@@ -73,7 +73,8 @@ const BlueOutlineButton = withStyles({
     height: '45px',
     padding: '10px 21px',
     marginTop: 'auto',
-    color: '#016EE3',
+    // color: '#016EE3',
```
Comment (cfernando415) [inline src/components/Home/Home.js:76]: remove comment

File: src/components/Home/Home.js [hunk 2]
```diff
@@ -677,7 +678,7 @@ export default function Home() {
               <Typography
                 {...{
                   'aria-label': 'Register for webmaking workathon',
-                  'aria-required': true,
+                  // 'aria-required': true,
```
Comment (cfernando415) [inline src/components/Home/Home.js:681]: remove commented code

**Review by cfernando415 — CHANGES_REQUESTED** [review]
pls see comment

File: src/components/Home/Home.js (hunk 1 above)
Comment (21shield) [reply src/components/Home/Home.js]: Resolved in commit 5bbf2b7

File: src/components/Home/Home.js (hunk 2 above)
Comment (21shield) [reply src/components/Home/Home.js]: Updated code in commit daf3c52

**Review by cfernando415 — APPROVED** [review]
:+1:

**Pull request merged**